fs.go

```diff
@@ -345,23 +364,33 @@ func (fs *FS) initRequestHandler() {
 	if cacheDuration <= 0 {
 		cacheDuration = FSHandlerCacheDuration
 	}
-	compressedFileSuffix := fs.CompressedFileSuffix
-	if len(compressedFileSuffix) == 0 {
-		compressedFileSuffix = FSCompressedFileSuffix
+
+	compressedFileSuffixes := fs.CompressedFileSuffixes
+	if len(compressedFileSuffixes["br"]) < 2 || !strings.HasPrefix(compressedFileSuffixes["br"], ".") ||
+		len(compressedFileSuffixes["gzip"]) < 2 || !strings.HasPrefix(compressedFileSuffixes["gzip"], ".") ||
+		compressedFileSuffixes["br"] == compressedFileSuffixes["gzip"] {
+		compressedFileSuffixes = FSCompressedFileSuffixes
+	}
+
+	if len(fs.CompressedFileSuffix) > 1 && strings.HasPrefix(fs.CompressedFileSuffix, ".") {
+		compressedFileSuffixes["gzip"] = fs.CompressedFileSuffix
+		compressedFileSuffixes["br"] = strings.TrimSuffix(fs.CompressedFileSuffix, filepath.Ext(fs.CompressedFileSuffix)) + ".br"
```

Comment:
Just default it to `FSCompressedFileSuffixes["br"]` instead of trying to come up with something clever.